crates/bevy_app/src/schedule_runner.rs

```diff
@@ -3,7 +3,10 @@ use crate::{
     plugin::Plugin,
     PluginsState,
 };
-use bevy_utils::{Duration, Instant};
+use bevy_utils::Duration;
+
+#[cfg(any(target_arch = "wasm32", feature = "std"))]
+use bevy_utils::Instant;
```

Comment:
I'm not a fan of how we bring this `Instant` in. I think it'd be worth considering some kind of `bevy_platform_support` crate which moves things like `spin`, `portable-atomic`, `critical-section`, `wasm-time`, etc. into a single spot. It'd be more focussed than `bevy_utils`, and possibly clean up some of the `cfg(...)` noise we have across Bevy.

Reply:
Yes please!